.github/workflows/lint-golang.yml

```diff
@@ -43,16 +46,16 @@ jobs:
       env:
         GH_ACCESS_TOKEN: ${{ secrets.ACTIONS_BOT_TOKEN }}
       run: git config --global "url.https://$GH_ACCESS_TOKEN@github.com/.insteadOf" https://github.com/
-  
+
     - name: go mod tidy (cluster-to-redpanda-migration)
       working-directory: src/go/cluster-to-redpanda-migration
       run: |
-        GOPRIVATE=github.com/redpanda-data/flux-controller-shim go mod tidy
+        GOPRIVATE=github.com/redpanda-data/* go mod tidy
```

Comment:
Why is this being set to `*` and (presumably) overriding the global env of `flux-controller-shim`? Can we just rely on the global setting and set it to `*` if need be?

Reply:
we are going to pull various internal repos from redpanda-data, this keeps us from explicitly having to list each one directly.

Reply:
not a blocker but can't we set line 16 to `*` instead of each one down here?

Reply:
im not sure since action env vars have been tricky for me. But good catch on 16. Ill have that fix in the next PR.